opentracing/src/main/java/opentracing/Span.java

```diff
@@ -54,10 +45,17 @@
   Span setTag(String key, boolean value);
 
   /** Same as {@link #setTag(String, String)}, but for numeric values. */
-  // numbers kindof suck.. we've no idea if this is a float, how many bits, etc.
   Span setTag(String key, Number value);
 
-  Span setTraceAttribute(String key, String value);
+  /**
+   * Set a Baggage item, represented as a simple string:string pair.
+   *
+   * Note that newly-set Baggage items are only guaranteed to propagate to future children of the given Span.
+   */
+  Span setBaggage(String key, String value);
```

Comment:
`setBaggageItem` per the spec and other languages... the "Baggage" itself is a set.